config/samples/apps_v1alpha1_nimcache.yaml

```diff
@@ -6,4 +6,25 @@ metadata:
     app.kubernetes.io/managed-by: kustomize
   name: nimcache-sample
 spec:
-  # TODO(user): Add fields here
+  source:
+    ngc:
+      modelPuller: nvcr.io/nim/meta/llama3-8b-instruct:1.0.0
+      pullSecret: ngc-secret
+      authSecret: ngc-api-secret
+      model:
+        profiles: []
+        autoDetect: true
+        precision: "fp8"
+        engine: "tensorrt_llm"
+        qosProfile: "throughput"
+        gpus:
+          product: "l40s"
+          ids:
+            - "26b5"
+        tensorParallelism: "1"
```

Comment:
FWIW, Meenakshi flagged me for having `1` in this field.  Some concern that readers might think it's necessary to specify `1` rather than omit for `1`.  Clearly up to you.

Reply:
We still need users to specify this for filtering. Otherwise we will end up pulling all models for `l40s` in this example those that will need `4` and `8` GPUs as well. This just makes it explicit on what user needs us to cache.